rmg.py

```diff
@@ -82,6 +82,10 @@ def parse_command_line_arguments(command_line_args=None):
     parser.add_argument('-t', '--walltime', type=str, nargs=1, default='00:00:00:00',
                         metavar='DD:HH:MM:SS', help='set the maximum execution time')
 
+    # Add option to select max number of processes for reaction generation
+    parser.add_argument('-n', '--maxProc', type=int, nargs=1, default=1,
+                        help='used by multiprocessing in react.py')
```

Comment:
This may not be the most helpful info, since users may not know what react.py is. Perhaps `max number of processors to use during reaction generation` or something similar.

Reply:
Done.